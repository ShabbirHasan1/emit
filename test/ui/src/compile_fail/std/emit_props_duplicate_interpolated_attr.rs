fn main() {
    emit::emit!("template {#[emit::optional] x}", x: 42);
}
