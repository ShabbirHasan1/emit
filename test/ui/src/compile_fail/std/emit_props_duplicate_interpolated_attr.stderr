error: keys that exist in the template and extra pairs can only use attributes on the extra pair
 --> src/compile_fail/std/emit_props_duplicate_interpolated_attr.rs:2:28
  |
2 |     emit::emit!("template {#[emit::optional] x}", x: 42);
  |                            ^^^^^^^^^^^^^^^^^^^
